src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -2537,13 +2537,98 @@ void M()
 ";
             var expectedDiagnostics = DiagnosticDescription.None;
 
-            // PROTOTYPE(dataflow): should we even have assignments for const locals? the symbol itself has a constant value.
             string expectedFlowGraph = @"
 Block[B0] - Entry
     Statements (0)
     Next (Regular) Block[B1]
-Block[B1] - Exit
-    Predecessors: [B0]
+        Entering: {R1}
+
+.locals {R1}
+{
+    Locals: [System.Int32 d]
+    Block[B1] - Block
+        Predecessors: [B0]
+        Statements (1)
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Int32, IsImplicit) (Syntax: 'd = 1')
```

Comment:
I'm rather conflicted about this change. There is no assignment in the IL, all references to the constant are replaced by the constant value. We even do this in the case of something like `const int i = true ? 1 : 2;`. I don't think we should be including these in the control flow graph unless the constant actually doesn't have a constant value, ie in error scenarios where there's a conditional that the compiler hasn't determined to be constant (`const int i = boolParam ? 1 : 2`).

Reply:
I will let the compiler team decide how much we want the CFG to match the IL. I agree with @AlekseyTs that whatever we do here, we must be consistent across fields and locals, and currently we are not.

Personally, I feel order or set of operations in CFG does not need to match the IL, it should just represent the control flow semantically as per users source code, and attempting to match IL every time just pushes the compiler optimizations into CFG. We should look at the IL just as a guidance to prevent us from generating incorrect CFG.